frontend/src/scenes/session-recordings/playlist/SessionRecordingPreview.tsx

```diff
@@ -298,14 +315,6 @@ export const SessionRecordingPreview = memo(
                                             <span>{recording.keypress_count}</span>
                                         </span>
                                     </Tooltip>
-                                    {recording.recording_ttl && (
```

Comment:
the preview was really busy with this on some screen layouts, so i thought we'd just show it when sorting by expiry